  Fix compilation with gcc 4.5

  [FBreader Ticket #340] -- http://old.fbreader.org/mantis/view.php?id=340

  Without that patch we have:
    Compiling Library.o ...Library.cpp:42:1: error: 'Library::Library' names the constructor, not the type

  Upstream-Status: Accepted [Fixed in Version 0.14.0]

--- fbreader-0.12.10-orig/fbreader/src/library/Library.cpp	2010-04-01 15:14:24.000000000 +0200
+++ fbreader-0.12.10/fbreader/src/library/Library.cpp	2010-04-13 13:34:06.000000000 +0200
@@ -39,7 +39,7 @@
 shared_ptr<Library> Library::ourInstance;
 const size_t Library::MaxRecentListSize = 10;
 
-Library::Library &Library::Instance() {
+Library &Library::Instance() {
 	if (ourInstance.isNull()) {
 		ourInstance = new Library();
 	}
